### PR TITLE
Switch to dedicated key

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,10 @@
-V=20220223
+V=20241216
 
 PREFIX = /usr/local
 
 install:
 	install -dm755 $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
-	# install -m0644 asahi-alarm{.gpg,-trusted,-revoked} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
-	install -m0644 asahi-alarm{.gpg,-trusted} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
+	install -m0644 asahi-alarm{.gpg,-trusted,-revoked} $(DESTDIR)$(PREFIX)/share/pacman/keyrings/
```
Comment (mkurz) [inline Makefile:7]: Why removed the `-revoked`?

File: master-keyids [hunk 2]
```diff
@@ -1 +0,0 @@
-1F35A37F9D85CBC2B74F595E37CE6162C969C50E joske
```
Comment (mkurz) [inline master-keyids:1]: Both files `master-keyids` and `master/*.asc` are currently not used by the `Makefile`.  Also it seems the `update-keys` script was not really used(?)

File: Makefile (hunk 1 above)
Comment (joske) [reply Makefile:7]: because the file doesn't exist (we don't have revoked keys). We could consider revoking marcan's key

Comment (mkurz) [reply Makefile:7]: But does installation fail when there is no file? I think it's ok to just have an empty file as well

File: master-keyids (hunk 2 above)
Comment (joske) [reply master-keyids:1]: yeah, not sure why those are created and exactly how this was supposed to work.

Comment (mkurz) [reply master-keyids:1]: I think the script was wip and never finished

File: Makefile (hunk 1 above)
Comment (joske) [reply Makefile:7]: yes `make install` fails if the file doesn't exist

Comment (mkurz) [reply Makefile:7]: I pushed an empty revoked file now.
(If you check current `/usr/share/pacman/keyrings/asahilinux-revoked` is also empty)

**Pull request merged**